konflux-ci/ui/core/proxy/nginx.conf

```diff
@@ -102,18 +102,6 @@ http {
             # nginx auth_request includes headers but not body
             proxy_set_header Content-Length   "";
             proxy_pass_request_body           off;
-
-            # See https://github.com/kubernetes/kubernetes/pull/47740 for information
```

Comment:
this is the only section that should be removed, we should keep all the rest.
The rest of the configuration are used for authenticating to the kube api, this is what gives nginx the permission to impersonate users when forwarding requests.